Libplanet.Tests/Blockchain/BlockChainTest.cs

```diff
@@ -1098,16 +1101,19 @@ HashDigest<SHA256>[] ListStateReferences(Address address)
             Assert.Empty(
                 store.Logs.Where(l => l.Item1 == "StoreStateReference")
             );
-            foreach (Block<DumbAction> block in blocks.Take(blocks.Length - 1))
+
+            for (var i = 0; i < chain.Count - 1; i++)
```

Comment:
This pattern seems too inconvenient.  How about having a method like `.EnumerateBlocks()`?  It would be good if it can be named simply `Blocks`, but it's already used.  In the long run, the current `Blocks` property would be better if it's renamed to like `BlockPool` or `AllBlocks`.

Reply:
I restored `BlockHashes` property to iterate block hashes instead of the blocks.